libraries/lib-telemetry/details/GoogleAnalyticsUA.h

```diff
@@ -0,0 +1,45 @@
+/*!********************************************************************
+
+ Audacity: A Digital Audio Editor
+
+ @file GoogleAnalyticsUA.h
+ @brief Declare a class for reporting telemetry events to Google Universal Analytics.
+
+ Dmitry Vedenko
+ **********************************************************************/
+
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+#include "ITelemetryService.h"
+
+namespace audacity
+{
+namespace telemetry
+{
+namespace details
+{
+class CommonHeaders;
+
+class GoogleAnalyticsUA final : public ITelemetryService
+{
+public:
+	explicit GoogleAnalyticsUA (std::string trackingID, const CommonHeaders* commonHeaders) noexcept;
```

Comment:
Tabs -> spaces, as per the Audacity coding standards docs. Also at other places in this file.
(Please don't start a holy war about which one is better.)

Reply:
Yeah, sure, a holy war about tabs and spaces. Let's go with that

Reply:
You are giving a review to code that will never reach production.

Reply:
I'm just trying to show how bad quality it is.